charts/alfresco-common/templates/_helpers-nginx.tpl

```diff
@@ -0,0 +1,24 @@
+{{/*
+Define annotations as provided in values
+*/}}
+{{- define "alfresco-common.nginx.annotations" }}
+{{- range $annotation, $value := .ingress.annotations }}
+  {{- if ne $annotation "nginx.ingress.kubernetes.io/server-snippet" }}
+  {{- $annotation | nindent 4 }}: |-
+    {{- $value | nindent 6 }}
+  {{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Define required annotations for secure nginx ingress
+We're overridong here ANY server snippet annotation and applying only ours trusted ones
+*/}}
+{{- define "alfresco-common.nginx.secure.annotations" }}
+{{- template "alfresco-common.nginx.annotations" . }}
```

Comment:
not totally sure it's a good idea to include the generic template here. What if we decide later that we need another template which add their own annotations? Better to keep also this one independent from the generic annotations template?